packages/components/src/presentation-components/properties/NavigationPropertyTargetSelector.tsx

```diff
@@ -159,7 +159,7 @@ function TargetSelectIndicatorsContainer<TOption extends OptionTypeBase>({ child
 }
 
 function TargetSelectDropdownIndicator<TOption extends OptionTypeBase>(_: IndicatorProps<TOption>) {
-  return <span className="iui-end-icon iui-actionable" >
```

Comment:
We probably need to use color variable that is used in [`iui-select` class](https://github.com/iTwin/iTwinUI/blob/bfe66dd80d26c119b7f605a7d57a83b207c12928/packages/itwinui-css/src/select/select.scss#L22) in order for this component to correctly work in different themes.

Reply:
fixed

Reply:
Also react-select component used here has [`styles`](https://react-select.com/styles#the-styles-prop) property that allows to set css styles on various inner components. I think it should be used for this `dropdownIndicator` too.